### PR TITLE
A few bits of back and suggestions

### PR DESCRIPTION
**Pull request by shonfeder**
Thanks for inviting me to give it a read! 

For lack of a better mechanism, I've left some comments in the body of the text.

### CODE REVIEW

File: content/posts/bitcoin.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ confused, including Elon Musk himself.
 Tesla, a bastion of our clean energy future, is helping us transition
 from fossil fuels to electricity.
 Bitcoin, a speculative digital asset, is using more energy 
-than Switzerland and is (or at least was!) largely powered by dirty Chinese coal. 
+than Switzerland and is (or at least was!) largely powered by dirty coal mined in China.
```
Comment (shonfeder) [inline content/posts/bitcoin.md:14]: Unless you plan to address cynophobia, I'd avoid that phrasing.

File: content/posts/bitcoin.md [hunk 2]
```diff
@@ -38,26 +38,26 @@ beneficial exchange to drive progress. But the modern incarnation of military-in
 monopolistic, American consumerism is not particularly nurturing for a sustainable future. 
 Tesla is perhaps American capitalism's best foot forward.
 
-Two is the supply chain of modern technology.
```
Comment (shonfeder) [inline content/posts/bitcoin.md:41]: Suggested these just cause the sequence `One, Two, Third` struck me as off. An alternative would be `First, Second, Third` or `One, Two, Three`.

File: content/posts/bitcoin.md [hunk 3]
```diff
@@ -38,26 +38,26 @@ beneficial exchange to drive progress. But the modern incarnation of military-in
...
-critical to sustainability. To the extent we have cars they should probably be electric. 
+critical to sustainability. To the extent that we have cars, they should probably be electric. 
 But more than anything, we need less cars. Denmark has the right idea. As does
 anyone who's ever gotten a street closed to cars, for any duration. Respect.
 
 Now I'm not trying to diss Tesla, per se. 
 Tesla is the beacon of a research and development process that is itself critical to
-sustainability. And if we want to make progress in batteries, which we should, cars are a great
+sustainability. <!-- I don't follow the reasoning of the next sentence: ---> And if we want to make progress in batteries, which we should, cars are a great
 way to go. But the fundamental problems we're facing today don't seem so much technical as they
```
Comment (shonfeder) [inline content/posts/bitcoin.md]: That is: why are cars a good way to make progress on batteries? Seems like those are orthogonal.

File: content/posts/bitcoin.md [hunk 4]
```diff
@@ -81,7 +81,7 @@ and is under consideration for legal tender status in a number of other South Am
 
 But Bitcoin isn't a precious metal. And it's not something produced by a company, nor a government, 
 nor really any distinctly identifiable organization.
-Bitcoin is a collective. An open-source, verifiable, transparent, decentralized, 
+Bitcoin is an open-source, verifiable, transparent, decentralized,
```
Comment (shonfeder) [inline content/posts/bitcoin.md]: iiuc, companies and governments are also collectives, and since you use the qualified "collective" in the next sentence...

File: content/posts/bitcoin.md [hunk 5]
```diff
@@ -170,6 +171,7 @@ the security of the Bitcoin network via Proof-of-Work (PoW) mining.
 Proof-of-Work entails a specific kind of cryptographic computation resembling
 a numerical puzzle that is difficult to solve but easy to verify. If
 computing power were evenly distributed throughout humanity (which of course
+<!-- why "could never be"? -->
 it's not and could never be), Bitcoin's Proof-of-Work mining would be a sort of Universal Basic Job,
```
Comment (shonfeder) [inline content/posts/bitcoin.md]: I think "it's not" probably suffices, and you don't lose people who think it is possible to have an equitable distribution of resources.

File: content/posts/bitcoin.md [hunk 6]
```diff
@@ -304,7 +306,7 @@ in the end for the better.
 
 Many people have heard the phrase "blockchains, not Bitcoin", 
 meant to express that blockchain technology is a meaningful innovation,
-but Bitcoin can be largely ignored. Of course we don't believe Bitcoin can or
+but Bitcoin can be largely ignored. Of course, I don't believe Bitcoin can or
```
Comment (shonfeder) [inline content/posts/bitcoin.md:307]: Just cause I didn't generally see third person used in this case of case early in the piece.

File: content/posts/bitcoin.md [hunk 7]
```diff
@@ -356,6 +358,7 @@ objective, international sources of trust, to the more subjective and local. Fro
 the cryptographically verifiable energy expenditure in a remote data center that
 is today's Proof-of-Work, to the communally verifiable delivery of life sustaining care work that is tomorrow's Proof-of-Care and [Proof-of-Plant][proof-of-plant]. 
 Proof-of-Stake may be the next step in that journey - somewhat more subjective, less anonymous, more socially aware than Proof-of-Work. 
+<!-- PoW has the same problems, right? With more bitcoin you can own more computer, which means you can amass and cointrol more bitcoin? -->
```
Comment (shonfeder) [inline content/posts/bitcoin.md]: Or else you could maybe give a line pointing indicating the different level at which you're identifying this tendency in the case of PoS vs. PoW.

File: content/posts/bitcoin.md (hunk 3 above)
Comment (ebuchman) [reply content/posts/bitcoin.md]: ah, just that they provide a good socioeconomic impetus for battery. car gas tanks are one of the most dominant energy stores in peoples experience so from a product-market perspective, cars are probably a good place battery innovation to get funded and delivered

File: content/posts/bitcoin.md [hunk 8]
```diff
@@ -38,26 +38,26 @@ beneficial exchange to drive progress. But the modern incarnation of military-in
...
 particularly conducive to the more human-scale and bio-diverse patterns that are 
-critical to sustainability. To the extent we have cars they should probably be electric. 
+critical to sustainability. To the extent that we have cars, they should probably be electric. 
 But more than anything, we need less cars. Denmark has the right idea. As does
 anyone who's ever gotten a street closed to cars, for any duration. Respect.
 
 Now I'm not trying to diss Tesla, per se. 
 Tesla is the beacon of a research and development process that is itself critical to
-sustainability. And if we want to make progress in batteries, which we should, cars are a great
+sustainability. <!-- I don't follow the reasoning of the next sentence: ---> And if we want to make progress in batteries, which we should, cars are a great
```
Comment (ebuchman) [inline content/posts/bitcoin.md]:
```suggestion
sustainability. And if we want to make progress in batteries, which we should, cars are a great
```

File: content/posts/bitcoin.md (hunk 3 above)
Comment (ebuchman) [inline content/posts/bitcoin.md]:
```suggestion
vehicle to ground that R&D. But the fundamental problems we're facing today don't seem so much technical as they
```

File: content/posts/bitcoin.md (hunk 4 above)
Comment (ebuchman) [reply content/posts/bitcoin.md]:
```suggestion
Bitcoin is a collective. An open-source, verifiable, transparent, decentralized, 
```

Sure but I'm trying to use it here more for its amorphous/illegible qualities as distinct from well bounded/definable "organizations"

File: content/posts/bitcoin.md [hunk 9]
```diff
@@ -111,8 +111,9 @@ wrought by the global financial system - they have thus dramatically betrayed th
 
 At least some of this appears due to a collapse in the literal values of money as a
 representative media. Unmoored from any real world process, 
+<!-- I like alliteration, but this is a lot of it and I tripped up a bit this clause :) -->
```
Comment (ebuchman) [inline content/posts/bitcoin.md]: Hah thanks, I can't help my self sometimes, gonna let this one lie.
```suggestion
```

File: content/posts/bitcoin.md [hunk 10]
```diff
@@ -146,15 +147,15 @@ contexts - it's being increasingly used to pay salaries, to make international p
...
 Of course it's also being used increasingly in ransomware payments and as fodder
 for speculative finance. In ransomwhere, it tragically illuminates the pervasive insecurity 
 and fragility of the systems underlying our critical infrastructure. In finance, it fuels the same kind
 of [leveraged and nebulous behaviour][tether] as the rest of the [shadow banking] system.
 Perhaps the greatest risk to Bitcoin is that it is [swallowed up by Wall
 St][epsilon-theory-bitcoin].
 
-But Bitcoin is not alone the solution to the world's monetary problems. 
-It may be necessary, but certainly not sufficient.
+It may be necessary, but it's certainly not sufficient.
```
Comment (ebuchman) [inline content/posts/bitcoin.md]:
```suggestion
But Bitcoin is not alone the solution to the world's monetary problems. It may be necessary, but it's certainly not sufficient.
```

File: content/posts/bitcoin.md [hunk 11]
```diff
@@ -170,6 +171,7 @@ the security of the Bitcoin network via Proof-of-Work (PoW) mining.
 Proof-of-Work entails a specific kind of cryptographic computation resembling
 a numerical puzzle that is difficult to solve but easy to verify. If
 computing power were evenly distributed throughout humanity (which of course
+<!-- why "could never be"? -->
```
Comment (ebuchman) [inline content/posts/bitcoin.md]:
```suggestion
```

File: content/posts/bitcoin.md (hunk 5 above)
Comment (ebuchman) [inline content/posts/bitcoin.md]:
```suggestion
it's not), Bitcoin's Proof-of-Work mining would be a sort of Universal Basic Job, 
```

File: content/posts/bitcoin.md [hunk 12]
```diff
@@ -284,7 +286,7 @@ will ebb and flow, like any great technology, simultaneously threatening both
 destruction and salvation of the human species. We have only to ask ourselves,
 "Is the game worth the candle?". And in this sense, the world faces much bigger
 problems than the energy usage and e-waste of Bitcoin. I think we'd do better to face those problems with
-Bitcoin at our back than without.
+Bitcoin at our back than opposing it.
```
Comment (ebuchman) [inline content/posts/bitcoin.md]:
```suggestion
Bitcoin at our back than opposing it.
```
```suggestion
Bitcoin at our back than without.
```

File: content/posts/bitcoin.md (hunk 3 above)
Comment (shonfeder) [reply content/posts/bitcoin.md]: That makes sense. So, iiuc, the reasoning is something like: given the current way we motivate and incentivize research, increasing the demand for, and so the profits to be gained from, electronic cars, may be the best bet to getting research directed to battery technology? I see how this could parallel the argument about bitcoin demand driving green tech innovation.
